src/main/java/com/google/sps/servlets/TopChartsCluster.java

```diff
@@ -0,0 +1,66 @@
+
+package com.google.sps.servlets;
+import java.util.ArrayList;
+
+public class TopChartsCluster {
+    String title = "Top Charts";
+    String backgroundImage = "background-image.png";
```

Comment:
Optional: can you actually add an image to `src/main/webapp/` and make this a url here to link to that image?